src/pages/product/tech-insights.js

```diff
@@ -0,0 +1,293 @@
+import has from 'lodash/has';
+import { SEO, SitewideFooter, SitewideHeader, Button, Headline } from 'components';
+import { AlternatingFeatureBlock, AlternatingFeatureWrapper } from 'components/landing';
+import Title from '../../components/Title';
+
+import TechInsightsScreenshotImg from '../../../content/assets/product-pages/techinsights-hero.svg';
+import CreateCheckImg from '../../../content/assets/product-pages/techinsights-create-check.svg';
+import IntegrationsImg from '../../../content/assets/product-pages/techinsights-integrations.png';
+import GitHubBotImg from '../../../content/assets/product-pages/techinsights-github-bot.svg';
+import Logi4jImg from '../../../content/assets/product-pages/techinsights-logi4j.svg';
+
+import { LOGOS } from '../../components/landing/CustomerLogoCloud';
+import { Tab, TabList, TabPanel, Tabs } from 'react-tabs';
+import classNames from 'classnames';
+
+const SEO_TITLE = 'Tech Insights: scorecards for Backstage [coming soon]';
+const LEAD = `Roadie’s Backstage-based Tech Insights lets you measure and improve software quality`;
+
+const PRODUCT = {
+  features: [
+    {
+      title: 'Raise the bar',
+      description: 'Create a culture of quality and accountability',
+      illustration: {
+        png: CreateCheckImg,
+        alt: 'A screenshot of an interface where a user is creating an automated check to determine if software is registered in PagerDuty.',
+      },
+      paragraphs: [
+        'Define best practices for software quality in your organization, and quickly find software that is not meeting expectations.',
+        `Roadie's automated checks can find software that is not receiving adequate support and maintenance, and teams who need help.`,
+      ],
+    },
+    {
+      title: 'Completely flexible. Integrates with anything',
+      description:
+        'Define checks against code, third-party APIs, and more...',
+      illustration: {
+        png: IntegrationsImg,
+        alt: '',
+      },
+      paragraphs: [
+        `Create automated checks that integrate with the tools you already use, like PagerDuty, Datadog, and GitHub. Build your own integration if we haven't supplied it.`,
+        'Create checks against source code to verify minimum library versions, ensure CI is configured, or require README files to exist.',
+      ],
+    },
+    {
+      title: 'Drive improvements over time',
+      description: 'Nudge teams towards building better software',
+      comingSoon: true,
+      illustration: {
+        png: GitHubBotImg,
+        alt: '',
+      },
+      paragraphs: [
+        'Set expectations around software quality through gamification. Let teams see their quality level vs. the teams around them.',
+        'Gently push teams towards imporovment with prompts in Slack channels and on pull requests.',
+      ],
+    },
+    {
+      title: 'Run migrations with ease',
+      description: 'Roadie gives you insights into library or API migrations',
```

Comment:
I guess nitpick but the rest of the subtitles have an imperative conjugation except this one.